webapp/components/PostCard/index.vue

```diff
@@ -123,6 +132,26 @@ export default {
         this.$toast.error(err.message)
       }
     },
+    calculateItemHeight() {
```

Comment:
@alina-beck do you still need this? The build server is crashing during frontend testing and it's trying to set some `style` attribute of `null`.